selection/main.py

```diff
@@ -46,6 +49,12 @@ def main(
 
     """
```

Comment:
should we add a warning if language is unspecified, or not `in ["en", "id", "pt]`)? or should we make language a required argument (i.e., remove the default)? also fine to keep it as-is, your call!